arbitrator/wasm-libraries/user-host-trait/src/lib.rs

```diff
@@ -477,6 +477,27 @@ pub trait UserHost: GasMeteredMachine {
         trace!("account_codehash", self, address, hash)
     }
 
+    fn account_code(&mut self, address: u32, dest: u32, size: u32) -> Result<(), Self::Err> {
+        self.buy_ink(HOSTIO_INK + EVM_API_INK)?;
```

Comment:
Let's also require enough gas for `size` bytes and pass `size` to Geth, which should only copy up to that many